parlai/core/agents.py

```diff
@@ -287,6 +287,9 @@ def load_agent_module(opt):
                 print("[ warning: overriding opt['" + str(k) + "'] to " + str(v) +
```

Comment:
actually can you change the loop to `for k, v in opt['override'].items()`? a little cleaner that way